server/sonar-db-dao/src/main/java/org/sonar/db/webhook/WebhookDao.java

```diff
@@ -28,13 +29,16 @@
 import org.sonar.db.organization.OrganizationDto;
 
 import static com.google.common.base.Preconditions.checkState;
+import static java.util.Collections.emptyList;
 
 public class WebhookDao implements Dao {
 
   private final System2 system2;
+  private final WebhookDeliveryDao webhookDeliveryDao;
 
-  public WebhookDao(System2 system2) {
+  public WebhookDao(System2 system2, WebhookDeliveryDao webhookDeliveryDao) {
```

Comment:
In our current state a DAO should not depend on another DAO. Callers of delete() methods should also call WebhookDeliveryDao#delete() methods.